test/behavior/tendrl/cluster_unmanaged.feature

```diff
@@ -0,0 +1,129 @@
+Feature: CLUSTER_UNMANAGED
+
+  Scenario: User sees cluster lists page after logging in
+    Given User opens the Tendrl link
+     When User passes correct login information
+     Then User should see "cluster list" page
+
+  Scenario: User sees cluster is unmanaged
+    Given User is on the "cluster list" page
+      And has at least one cluster available
+     Then User should see "No" under Managed
+
+  Scenario: User sees cluster status icon
+    Given User is on the "cluster list" page
+      And has at least one cluster available
+     When User hovers over cluster status icon
+      And User should see status is unknown
+
+  Scenario: User sees the cluster name/uuid
+    Given User is on the "cluster list" page
+      And has at least one cluster available
+     Then User should see UUID of unimported cluster
+
+  Scenario: User sees Cluster Version info
+    Given User is on the "cluster list" page
+      And has at least one cluster available
+     Then User should see Cluster Version info
+
+  Scenario: User sees hosts information
+    Given User is on the "cluster list" page
+      And has at least one cluster available
+     When User clicks on hosts link
+     Then User should be brought to hosts page
+
+  Scenario: Cluster is unavailable for import
```

Comment:
which scenario is this? can you explain

Reply:
I'm not sure in what other circumstances this situation arises, but if you unmanage the cluster you are unable to import it again until the task is complete. During this time the import button is greyed out and unusable. Not sure if this is a relevant scenario for us to test.
![unable_to_import](https://user-images.githubusercontent.com/11084059/46550629-a055c480-c8a3-11e8-90c8-5f30e88ced01.png)